src/interface/src/app/services/notes.service.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../environments/environment';
+export interface Note {
+  id: number;
+  user_id: number;
+  user_name: string;
+  content: string;
+  created_at: string;
+  can_delete?: boolean;
+}
+
+export type NotesModelName = 'planning_area' | 'project_area';
+
+export interface NoteModel {
```

Comment:
Do we need this interface? also `NoteModel` is kinda confusing with the actual note model on top